sim/core/buffs.go

```diff
@@ -1326,31 +1327,31 @@ func (raid *Raid) ProcReplenishment(sim *Simulation, src ReplenishmentSource) {
 	}
 }
 
-func TotemOfWrathAura(character *Character) *Aura {
+func TotemicWrathAura(character *Character) *Aura {
 	aura := character.GetOrRegisterAura(Aura{
-		Label:      "Totem of Wrath",
-		ActionID:   ActionID{SpellID: 57722},
+		Label:      "TotemicWrath",
+		ActionID:   ActionID{SpellID: 77746},
 		Duration:   NeverExpires,
 		BuildPhase: CharacterBuildPhaseBuffs,
 		OnReset: func(aura *Aura, sim *Simulation) {
 			aura.Activate(sim)
 		},
 	})
-	spellPowerBonusEffect(aura, 280)
+	spellPowerBonusEffect(aura, 0)
```

Comment:
Can probably remove the second argument to `spellPowerBonusEffect` since these now provide flat % SP buffs consistently